test/src/util/sugar.rs

```diff
@@ -0,0 +1,33 @@
+use crate::util::chain::forward_main_blocks;
+use crate::util::mining::{mine_until_out_bootstrap_period, mine_until_out_ibd_mode};
+use crate::Node;
+
+pub fn out_ibd_mode(nodes: &[Node]) {
+    if let Some(node0) = nodes.first() {
+        mine_until_out_ibd_mode(node0);
+        if nodes.len() <= 1 {
+            return;
+        }
+
+        let tip_number = node0.get_tip_block_number();
+        let range = 1..tip_number + 1;
+        for node in nodes.iter().skip(1) {
+            forward_main_blocks(node0, node, range.clone());
+        }
+    }
+}
+
+pub fn out_bootstrap_period(nodes: &[Node]) {
```

Comment:
I think `sugar` is not a clear enough name, move `out_bootstrap_period` into `mine.rs` and move `out_ibd_mode` into `node.rs` maybe a better way. If I want some function, I will find it in the relate file first, and maybe I'll never touch the sugar if I don't know what it has.

Reply:
Agree! I will move them two into `mine.rs`, which should be treated as mining-related suites.